### PR TITLE
option to store dia TOF calib directly from digits

### PR DESCRIPTION
**Pull request by noferini**

### CODE REVIEW

**Comment by noferini** [comment]
I needed to fix a missing info in the TOF digit reader (propagation of TFinfo to Diagnostics). Now it works as expected. Ready for the review

File: Detectors/TOF/workflowIO/src/TOFCalibWriterSpec.cxx [hunk 1]
```diff
@@ -55,7 +58,7 @@ DataProcessorSpec getTOFCalibWriterSpec(const char* outdef, bool toftpc, bool ad
                                 BranchDefinition<CalibInfosType>{InputSpec{"input", o2::header::gDataOriginTOF, ddCalib, 0},
                                                                  "TOFCalibInfo",
                                                                  "calibinfo-branch-name",
-                                                                 1,
+                                                                 !onlyDia,
```
Comment (shahor02) [inline Detectors/TOF/workflowIO/src/TOFCalibWriterSpec.cxx:61]: to be sure : do you want to disable this branch for `onlyDia = false`?

Comment (noferini) [reply Detectors/TOF/workflowIO/src/TOFCalibWriterSpec.cxx:61]: No, I want to disable it for onlyDia=true. Is it wrong?

Comment (shahor02) [reply Detectors/TOF/workflowIO/src/TOFCalibWriterSpec.cxx:61]: No, my question was inverted, should be ok.

**Review by shahor02 — APPROVED** [review]

**Pull request merged**